src/App.js

```diff
@@ -14,11 +15,62 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selected: '',
+    space: ' ',
+  };
+
+  saveHandler = (productName) => {
+    this.setState({ selected: productName });
+  }
+
+  clearSelected = () => {
+    this.setState({ selected: '' });
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <div className="selected">
+          <h1>
+            Selected good: -
+            {this.state.space + this.state.selected}
```

Comment:
I think you don't need 'this.state.spase', you could just use backticks
```suggestion
            {`Selected good: - ${this.state.selected}`}
```

Reply:
згоден з furude-rika, а навіть якщо тобі буде потрібна якась константа то state це не найкраще місце щоб її зберігати